ci/docker/install/ubuntu_python3.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# build and install are separated so changes to build don't invalidate
+# the whole docker cache for the image
```

Comment:
This file is mostly equivalent with https://github.com/apache/incubator-mxnet/blob/master/ci/docker/install/ubuntu_python.sh

Reply:
Oh, sorry for that then. Didn't see that discussion.

Could you just install the ubuntu_python.sh script? Even if you don't use Python2 (which I think you actually do because you install it one line further down in the Dockerfile), I don't think it hurts to install it. Or is there any problem that comes up if you simply run ubuntu_python.sh?

Reply:
ok